src/analysis/function_parameters.rs

```diff
@@ -198,12 +198,36 @@ pub fn analyze(
         .filter_map(|p| p.array_length.map(|pos| (pos, p)))
         .collect();
 
+    let mut to_remove = Vec::new();
+    let mut correction_instance = 0;
+
+    for par in function_parameters.iter() {
+        if par.scope.is_none() {
+            continue;
+        }
+        if let Some(index) = par.closure {
```

Comment:
Unfortunately the attribute is also (wrongly) on the `user_data` sometimes. When using it you should first check if the parameter is callback-typed, and if not just ignore the existence of the attribute.

Reply:
Hence why I added above `par.scope.is_none()` to ignore this case. ;)

Reply:
Ah clever. That works of course